
# This is a patch for Crypt-SSLeay-0.57.orig to update it to Crypt-SSLeay-0.57
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "Crypt-SSLeay-0.57.orig\Makefile.PL" "Crypt-SSLeay-0.57\Makefile.PL"
Index: ./Makefile.PL
*** ./Makefile.PL	Mon Sep 17 14:36:59 2007
--- ./Makefile.PL	Sat May 15 14:02:53 2010
***************
*** 18,23 ****
--- 18,24 ----
  );
  
  $opt_default ||= $ENV{CRYPT_SSLEAY_DEFAULT};
+ $opt_libpath ||= $ENV{OPENSSL_PREFIX};
  
  # FIND POSSIBLE SSL INSTALLATIONS
  
***************
*** 336,346 ****
      my $version;
      my $type;
      while (<VERSION_FILE>) {
!         if (/^#define\s+$version_match\s+0x0+(\d\d\d)/) {
              $version = $1;
!             $version =~ s/(\d)0(\d)/$1$2/;            
              $type = ($version > 92) ? "OpenSSL" : "SSLeay";
!             $version = join('.', split(//, "0$version"));
              last;
          }
      }
--- 337,347 ----
      my $version;
      my $type;
      while (<VERSION_FILE>) {
!         if (/^#define\s+$version_match\s+0x(\d\d\d\d\d)/) {
              $version = $1;
!             $version =~ s/(\d)0(\d)0(\d)/$1$2$3/;            
              $type = ($version > 92) ? "OpenSSL" : "SSLeay";
!             $version = join('.', split(//, "$version"));
              last;
          }
      }
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Sat May 15 14:03:31 2010
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "Makefile.PL" 10779 1273953773 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Sat May 15 14:03:31 2010] ####
#### Patch checksum: 73 2459 11951 ####
#### Checksum: 91 3159 4409 ####
